### DISCUSSION TITLE
Some features ideas

### DISCUSSION THREAD

**Discussion by blazing-ph0enix** [FossifyOrg/General-Discussion#531] [Ideas]
Greetings!

I've been in love with the Fossify phone app and it's really helpful in escaping google's ecosystem.

Besides carrier video calling support (Which I am not sure how hard it is to implement that) I would like to suggest a basic thing.

When I get someone's call, I get to see the caller's name and number both.
I would like it better if it shows only the name, because anyone around me can peek at my phone and get to see the number of the caller. It would be better if I could avoid these circumstances.

Besides that, everything is already beyond our expectations.

Loving the app!

Thank you for your efforts and time! 😇

**Comment by Aga-C** [comment]
> Besides carrier video calling support (Which I am not sure how hard it is to implement that)

It's already requested: https://github.com/FossifyOrg/Phone/issues/113.

> I would like it better if it shows only the name

Can you raise a [feature request](https://github.com/FossifyOrg/Phone/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml) for this? Thanks!

**Reply by blazing-ph0enix** [reply]
Oh okay, should I close this discussion? or delete it entirely? or just make a new feature request?

thanks!

**Reply by Aga-C** [reply]
Just make a new feature request 🙂
